GitUI/Editor/FileViewerInternal.cs

```diff
@@ -646,5 +647,30 @@ public TestAccessor(FileViewerInternal control)
 
             public TextEditorControl TextEditor => _control.TextEditor;
         }
+
+        public void SetGutterAvatars(List<GitBlameDisplay> avatars)
```

Comment:
Applying `AppSettings.BlameShowAuthorAvatar` looks to me like a second responsibility for the current method.
Wouldn't it be better to separate it out into a property `bool ShowGutterAvatars`?